package.json

```diff
@@ -13,7 +13,8 @@
     "eslint-plugin-prettier": "2.4.0",
     "materialize-css": "0.100.2",
     "prettier": "1.8.0",
-    "sass-lint": "1.12.1"
+    "sass-lint": "1.12.1",
+    "snyk": "1.68.0"
```

Comment:
If we are adding new dependencies, can we update the yarn.lock file?

 install and rebuild the yarn.lock file:
`yarn install`

`git add yarn.lock`